nomad/csi_endpoint_test.go

```diff
@@ -1100,6 +1100,7 @@ func TestCSIVolumeEndpoint_ListExternal(t *testing.T) {
 	// List external volumes; note that none of these exist in the state store
 
 	req := &structs.CSIVolumeExternalListRequest{
+		PluginID: "minnie",
```

Comment:
Note to reviewers: these tests _should_ have failed before and weren't because this bug masked the test bug.